app/code/Magento/Catalog/Block/Product/AbstractProduct.php

```diff
@@ -34,13 +34,6 @@ class AbstractProduct extends \Magento\Framework\View\Element\Template
      */
     protected $_defaultColumnCount = 3;
 
-    /**
-     * Product amount per row depending on custom page layout of category
-     *
-     * @var array
-     */
-    protected $_columnCountLayoutDepend = [];
```

Comment:
Please, mark property as deprecated due to backward compatibility instead of remove.

Reply:
Thanks Stas, somehow I missed it, while "do not eliminate any usages/functionality yet" looked not correct to me.